android/src/main/java/com/swmansion/reanimated/transitions/ChangeTransition.java

```diff
@@ -3,15 +3,15 @@
 import android.animation.Animator;
 import android.animation.AnimatorSet;
 import android.animation.TimeInterpolator;
-import android.support.annotation.NonNull;
-import android.support.annotation.Nullable;
-import android.support.transition.ChangeBounds;
-import android.support.transition.ChangeTransform;
-import android.support.transition.Transition;
-import android.support.transition.TransitionPropagation;
-import android.support.transition.TransitionValues;
 import android.view.ViewGroup;
 
+import androidx.annotation.Nullable;
+import androidx.transition.ChangeBounds;
+import androidx.transition.ChangeTransform;
```

Comment:
I wonder if we will be still able to use in old RN version

Reply:
I don't think this will be compatible with the old `RN` version. However, we can manually write a doc for those who use `react-native` version below `0.60`.